test/cases/010_platforms/000_qemu/001_run/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# SUMMARY: Test running an image with qemu
+# LABELS:
+# AUTHOR: Dave Tucker <dt@docker.com>
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+IMAGE_NAME=test-qemu-build
+
+clean_up() {
+	# remove any files, containers, images etc
+	rm -rf "${LINUXKIT_TMPDIR:?}/${IMAGE_NAME:?}*" || true
+}
+
+trap clean_up EXIT
+
+# Test code goes here
+[ -f "${LINUXKIT_TMPDIR}/${IMAGE_NAME}-kernel" ] || exit 1
```

Comment:
I don't really like how the images build by the previous test are used in this test. Tests should really be independent of each other because otherwise I can't run just run `platforms/qemu/test`